openedx/frontend-app-course-authoring
# feat: make placeholder depend on api response

The new Course Authoring MFEs are being rolled out to beta testers. Their access is dependent on a waggle flag for each page. To guarantee people who are not part of the beta testing group access the page, the get API call checks if the waffle flag is enabled. If the flag is not enabled, the API returns a 403 response. This response code will have the page render the page under construction component.

JIRA Ticket: [TNL-10916](https://2u-internal.atlassian.net/browse/TNL-10916)

Testing instructions

1. Navigate to course authoring's [advanced settings page](http://localhost:2001/course/course-v1:krisEdx+ka101+2023-01/settings/advanced). The page should say under construction.
2. In Django Admin, enable `contentstore.new_studio_mfe.use_new_advanced_settings_page`
3. Navigate back to course authoring's [advanced settings page](http://localhost:2001/course/course-v1:krisEdx+ka101+2023-01/settings/advanced). The page should render the list/form of advanced settings.


Other information
This change is dependent on edx-pladform [PR 32795](https://github.com/openedx/edx-platform/pull/32795). Use the branch in to connected PR for testing.
